testing/functional/tests/cypress/integration/03-ministry/05-student-note-restrictions.spec.ts

```diff
@@ -0,0 +1,575 @@
+import MinistryCustomCommand from "../../custom-command/ministry/MinistryCustomCommand";
```

Comment:
Are these tests part of the ticket?

Reply:
Yes, under custom commands all sections have common login cases defined.

<img width="293" alt="image" src="https://user-images.githubusercontent.com/98920571/167527048-4755ac5d-9fb1-49a1-a51c-23ff0c55c338.png">

Reply:
My question was more about the scope of the ticket. The tests on this file seems to be outside the scope of the ticket 😉